### ISSUE TITLE
Code style checks

### ISSUE THREAD

**Issue by Dartlexx** [epam/CoroutinesCache#9]
I think that this project would benefit if some Code Style checks would be used. Like KtLint and Detect.

Also, in several classes I noticed that nullable var field could be replaced with non-null lateinit or simple non-null field.

**_Record_** class is the main example of it:
```
    private var timeAtWhichWasPersisted: Long = 0
    private var dataClassName: String? = null
    private var dataCollectionClassName: String? = null
    private var dataKeyMapClassName: String? = null
```
These fields are changed only in **_init_** method.

**Comment by Dartlexx** [comment]
I have implemented code style checks in PR#21.
The rest of proposal will be implemented as separate PR, because it will be big enough and I don't want it to meddle with simple code style fixes.

**Comment by Dartlexx** [comment]
Most of those other issues were closed in previous or existing Pull Requests. Closing the issue
